File: part3/erlang/src/base/hello.erl
% hello world program
%% 声明的一般形式
%% 连字符（–）通常与模块，导入和导出语句一起使用。
%% 连字符用于为每个语句赋予相应的含义。
-module(hello).
-export([start/0]).

start() ->
  io:fwrite("Hello, world!\n").

File: part14/pyqt5/simple-tool-box/src/hello.py
import sys
from PyQt5.QtCore import *
from PyQt5.QtGui import *
from PyQt5.QtWidgets import *
def window():
   app = QApplication(sys.argv)
   w = QWidget()
   b = QLabel(w)
   b.setText("Hello World!")
   # x y width height
   w.setGeometry(100, 100, 280, 80) 
   w.move(60, 15)
   w.setWindowTitle("PyQt5 App")
   helloMsg = QLabel('<h1>Hello World!</h1>', parent=w)
   helloMsg.move(60, 15)
   w.show()
   sys.exit(app.exec_())
if __name__ == '__main__':
   window()